.changeset/late-pumas-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/http": minor
```

Comment:
I don't think we need to commit this changeset unless we're doing a version bump on one of the packages. The other 3 file updates look great tho!